app/src/main/res/values/strings.xml

```diff
@@ -66,6 +66,8 @@
     <string name="light_theme_title">Light</string>
     <string name="dark_theme_title">Dark</string>
     <string name="black_theme_title">Black</string>
+    <string name="device_dark_theme_title">Device theme (Dark)</string>
+    <string name="device_black_theme_title">Device theme (Black)</string>
```

Comment:
To be honest, when testing the APK, I did not know what the two settings were supposed to change.

We need to make clear that the dark or black theme is used when the night mode is enabled, light theme otherwise. However, I have no idea how to put that in a few words.

Reply:
I don't know neither. I've only seen this way when there is dark black and light theme, but I agree it's not ideal

Reply:
@Stypox @wb9688 @Poolitzer @opusforlife2 what do you think about that? ^

Reply:
I've seen on various apps: Use/Follow system/device theme.

So you could go with: Follow system theme (dark/black).

Alternatively, the "follow system theme" option could be a separate toggle which works only with Dark and Black radio buttons. FairEmail uses such a toggle.

Reply:
I really like the idea of switch.

Reply:
I like how GitHubs android app does it:
![Screenshot_20200606-142731](https://user-images.githubusercontent.com/25934244/83956873-2b285880-a817-11ea-86db-7bb1cbadeb88.png) ![Screenshot_20200606-142735](https://user-images.githubusercontent.com/25934244/83956871-2a8fc200-a817-11ea-9a44-427c94dfc044.png)

Reply:
I also prefer "Follow system theme (dark/black)"